core/iwasm/aot/aot_loader.c

```diff
@@ -3856,6 +3886,8 @@ aot_unload(AOTModule *module)
         bh_hash_map_destroy(module->ref_type_set);
     }
     os_mutex_destroy(&module->rtt_type_lock);
+    if (module->rtt_types)
+        wasm_runtime_free(module->rtt_types);
```

Comment:
Should also free every element?

```suggestion
    if (module->rtt_types) {
        uint32 i;
        for (i = 0; i < module->type_count; i++) {
            if (module->rtt_types[i])
                wasm_runtime_free(module->rtt_types[i]);
        }
        wasm_runtime_free(module->rtt_types);
    }
```

Reply:
Thanks, fixed, and fix issue in commit_params_to_frame_of_import_func.